### ISSUE TITLE
24Net cls_Layer

### ISSUE THREAD

**Issue by inspirit** [CongWeilin/mtcnn-caffe#7]
Hi, i see 2 methods in pythonLayer.py:
cls_Layer and cls_Layer24
roi_Layer and roi_Layer24

in prototxt you use 24 version, why do you need another version? is it used during training anywhere?
thank you

**Comment by CongWeilin** [comment]
cls_Layer is full convolution net
cls_Layer is full connection net
watch the last layer

**Comment by inspirit** [comment]
yes, so when do you use ```cls_Layer``` and when ```cls_Layer24```?
